src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Github integration
+tags:
+  - New Relic integrations
+  - Github integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+
+<img
+  title="Github Integration dashboard"
+  alt="Github Integration dashboard"
+  src="/images/github-integration-screenshot.webp"
+/>
+
+<figcaption>
+After setting up our Github integration, see your data of teams, repos and users right out of the box.
+</figcaption>
+
+
+## Set up the Github integration [#set-up-github]
+
+Complete the following steps to set up the Github integration:
+
+<Steps>
+<Step>
+
+## Integration Details [#integration-details]
+
+By default, the Power Service catalog is selected, which will ensure to import all the data from Teams, Repositories and Users.
```

Comment:
This lacks context. Where am I performing this? Also, as per the instruction, I don't really have to perform anything. It says Power Service catalog is the default selection. And if that's true indeed, then do we really need a step. All we need is to call out that the catalog must be selected for the setup.

Also, what's a Power Service catalog? Don't we need to explain that? Can we assume users already know this? If latter, then that should get called out as a prereq.

Reply:
The selection is by default for our MVP. But latter on we are planning to make that optional. So broadly speaking its not by default . I will update the text.